deploy/operatorsource.crd.yaml

```diff
@@ -2,12 +2,65 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: operatorsources.marketplace.redhat.com
+  annotations:
+    displayName: Operator Source
+    description: Represents an OperatorSource.
 spec:
   group: marketplace.redhat.com
   names:
     kind: OperatorSource
     listKind: OperatorSourceList
     plural: operatorsources
     singular: operatorsource
+    shortNames:
+    - os
```

Comment:
`os` or `opsrc`? Mostly to avoid an overloaded name like `os` which is also used as shorthand for `OpenShift`

Reply:
Done.